src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -988,9 +992,10 @@ def __call__(self, poller):
             self.cli_ctx.get_progress_controller().stop()
```

Comment:
```suggestion
            self.progress_bar.end()
```

Reply:
And I think `self.cli_ctx.get_progress_controller().begin()` should also be replaced by `self.progress_bar.begin()` instead of

Reply:
yes